en/deploy-on-aws-eks.md

```diff
@@ -262,12 +324,12 @@ This section describes how to scale out the EKS node group and TiDB components.
 
 ### Scale out EKS node group
 
-The following example shows how to scale out the `tikv` group of the `${clusterName}` cluster to 4 nodes:
+The following example shows how to scale out the `tikv-1a` group of the `${clusterName}` cluster to 2 nodes:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-eksctl scale nodegroup --cluster ${clusterName} --name tikv --nodes 4 --nodes-min 4 --nodes-max 4
+eksctl scale nodegroup --cluster ${clusterName} --name tikv-1a --nodes 2 --nodes-min 2 --nodes-max 2
```

Comment:
Scaling out needs to be evenly in 3 AZs for TiKV.

Reply:
Updated.